plugins/parsers/prometheusremotewrite/README.md

```diff
@@ -38,7 +36,36 @@ prompb.WriteRequest{
 
 ```
 
-**Example Output**
+### Example Output
 ```
 prometheus_remote_write,instance=localhost:9090,job=prometheus,quantile=0.99 go_gc_duration_seconds=4.63 1614889298859000000
 ```
+
+**For alignment with the [InfluxDB v1.x Prometheus Remote Write Spec](https://docs.influxdata.com/influxdb/v1.8/supported_protocols/prometheus/#how-prometheus-metrics-are-parsed-in-influxdb)**
+
+- Use the [Starlark processor](https://github.com/influxdata/telegraf/blob/master/plugins/processors/starlark/README.md) to rename the measurement name to the fieldname and rename the fieldname to value. Use namepass to only apply the script to prometheus_remote_write metrics
+
+- Example script: 
```

Comment:
maybe just link to the script so we don't have to keep the two docs in sync.

Reply:
yeah i suggested otherwise so people could just copy and paste from the readme but one extra click isn't a big deal